src/main/java/com/sandeep/demoemployee/entity/Designation.java

```diff
@@ -0,0 +1,53 @@
+package com.sandeep.demoemployee.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.persistence.*;
+
+@Entity
+public class Designation
+{
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    @JsonIgnore
+    private int dsgnId;
+    @Column
+    @JsonProperty(value = "jobTitle")
+    private String role;
+    @JsonIgnore
+    private float lvlId;
```

Comment:
Just a minor comment, Variable name would have been more good if it was levelId.